tests/test_oauth1_session.py

```diff
@@ -20,6 +25,20 @@
     unicode_type = unicode
     bytes_type = str
 
+# Monkey patch Python 2.6 unittest
+if not hasattr(unittest, 'SkipTest'):
+    unittest.SkipTest = RuntimeWarning
+    unittest.TestResult.real_add_error = unittest.TestResult.addError
+
+    def patched_addError(self, test, exc_info):
+        if exc_info[0] is RuntimeWarning:
+            print(str(exc_info[1]), end=' ', file=sys.stderr)
```

Comment:
So that the unittest output shows something useful. i.e.
https://travis-ci.org/jayvdb/requests-oauthlib/jobs/82236301#L176